OpenOversight/app/validators.py

```diff
@@ -1,10 +1,19 @@
 from us import states
+from urllib.parse import urlparse
 
 
 def state_validator(state):
     list_of_states = [st.abbr for st in states.STATES]
 
-    if state not in list_of_states and state != 'DC':
+    if state not in list_of_states and state != 'DC' and state is not None:
```

Comment:
Why do we need to handle the possibility that the `state` parameter is `None`?

Reply:
The models for `LicensePlate` and `Location` both allow their field `state` to be `NULL` that's why I want to allow it for the import. So I guess the only time it is relevant is when it's unknown which state a license plate belongs to. I am fine with reverting this to not allowing `None`